app/master/cluster_master.py

```diff
@@ -62,12 +63,15 @@ def api_representation(self):
             'slaves': slaves_representation,
         }
 
-    def builds(self):
+    def builds(self, start=None, amount=None):
```

Comment:
I mentioned this in your proposal, but just to push a bit harder 😛: I think I prefer "limit/offset" to "amount/start" just because there is precedent in other contexts, so might help understanding. I'm not sure they're objectively better words, but since "limit/offset" is used frequently in database contexts I feel like someone reading this code is more likely to be able to interpret those words quickly. 

Also, want to add type hints throughout? :smile_cat: